Makefile

```diff
@@ -26,7 +26,7 @@ itest: test
 	tox -e paasta_itests
 
 # See the makefile in yelp_package/Makefile for packaging stuff
-itest_%:
+itest_%_yelp:
```

Comment:
yes, this seems to work. I can run ``make itest_trusty_yelp`` and ``make itest_lucid_yelp`` and it does the right thing